src/backends/tensorrt/plan_backend.h

```diff
@@ -417,10 +417,15 @@ class PlanBackend : public InferenceBackend {
     // separate_output_stream is selected to overlap copy and execution safely.
     int next_buffer_binding_set_;
 
-    // The aer Context::num_expected_bindings_ number of IOBindingInfo elements
-    // for copy stream
+    // There are Context::num_expected_bindings_ number of IOBindingInfo
+    // elements for copy stream
     std::vector<std::vector<IOBindingInfo>> io_binding_infos_;
 
+    std::vector<std::tuple<
+        std::string, inference::DataType, std::vector<int64_t>, const char*,
+        TRITONSERVER_MemoryType, int64_t, BackendResponder*>>
+        processTensor_tuples_;
```

Comment:
You also must not use the processTensor_tuples_ as a class member variable. The plan backend context can issue multiple requests. The tuples will be over ridden by the latest request issue. This can be a part of the payload.